app/services/propublica_parser.rb

```diff
@@ -14,6 +14,22 @@ def parse_house_members
     end
   end
 
+  def update_representatives(representative)
+    if representative.first.class == HouseMember
+      reps_json = @service.get_house_members[:data]
+    else 
+      reps_json = @service.get_senators[:data]
+    end 
+
+    representative.all.each do |representative|
+      rep_data = reps_json.find.each { |rep| rep[:congress_id] == representative.congress_id }
+      rep_data[:last_updated] = rep_data[:last_updated].to_datetime
+      if rep_data.delete(:in_office)
+        representative.update(rep_data)
+      end
+    end 
+  end
+
```

Comment:
This method updates both house_members and senators and is called from the new rake task

Reply:
This is awesome